test/specs/scenarios/document.js

```diff
@@ -177,7 +177,7 @@ export const documentScenarios = async (lang) => {
         confirm.verifyUseAnotherFileError(copy)
       })
 
-      it('should return glare detected message on front and back of doc', async () => {
+      it('should return image quality message on front id card', async () => {
```

Comment:
Minor: IMO should be described as "front of doc". As it is it's slightly confusing that Driving Licence is selected but ID card is uploaded. Though there's not validation for matching doc type uploaded vs selected.